aten/src/ATen/native/cuda/GroupedBlas.cpp

```diff
@@ -669,9 +669,12 @@ std::optional<c10::ScalarType> out_dtype) {
   // _scaled_mm_allowed_device is used here within _grouped_mm_cuda which seems incorrect since scale is not used.
   // the _grouped_mm_fallback should be safe for any ROCm GPU since it's just calling typical mm/bmm
   bool use_fast_path = false;
+  // For windows ROCm, make sure use_fast_path is false
```

Comment:
`USE_ROCM_CK_GEMM` can also be disabled on Linux. This comment doesn't need to mention Windows. You should also be able to reproduce on Linux by turning off that option